src/main/java/com/dev/cinema/model/dto/CinemaHallRequestDto.java

```diff
@@ -1,9 +1,13 @@
 package com.dev.cinema.model.dto;
 
+import javax.validation.constraints.Size;
 import lombok.Data;
+import org.springframework.lang.NonNull;
 
 @Data
 public class CinemaHallRequestDto {
+    @NonNull
```

Comment:
you can add custom message - @NotNull(message = "...")

Reply:
It would be better :D

Reply:
Oh, unfortunantley i dont have this option with NonNull annotation :(